File: appQuadrado.py
from flask import Flask, jsonify

# iniciar o servidor Flask:
# python app.py

# Cria uma instância do Flask
app = Flask(__name__)

# Define uma rota para a página inicial
@app.route('/')
def index():
    return 'Hello World!'

# Define uma rota para a API
@app.route('/api/elevarAoQuadrado/<int:num>')
def quadrado(num):
    result = num ** 2
    return jsonify({'Elevado ao quadrado': result})

# Executa 
if __name__ == '__main__':
    app.run(debug=True)

File: appQuadrado.js
const express = require('express');
const app = express();

// Define uma rota para a página inicial
app.get('/', (req, res) => {
    res.send('Hello World!');
});

// Define uma rota para a API
app.get('/api/elevarAoQuadrado/:num', (req, res) => {
    const num = parseInt(req.params.num); // convertendo o valor do parâmetro "num" em um número inteiro
    const result = num ** 2;
    res.json({'Elevado ao quadrado': result});
});

// Inicia o servidor
app.listen(3000, () => {
    console.log('Servidor rodando na porta 3000!');
});

// http://localhost:3000/

// exemplo de URL: 
// http://localhost:3000/api/elevarAoQuadrado/5